app/src/forms/form/exportService.js

```diff
@@ -42,15 +42,49 @@ const EXPORT_FORMATS = Object.freeze({
 
 const service = {
 
+  _readSchemaFields: async (schema) => {
```

Comment:
Would it be fine to remove this async call? I dont believe this is making any await calls.